### PR TITLE
Review Project 1

### PR DESCRIPTION
**Pull request by oluwajuwon8**
This is a review project by microverse.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,6 +55,9 @@ <h1 id="title">Micheal Jackson</h1>
         </div>
         <a href="https://en.wikipedia.org/wiki/Michael_Jackson" id="tribute-link" target="_blank">more information</a>
     </div>
+    <!-- <div>
+        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Commodi impedit, nihil veniam modi laudantium, animi quia odio rerum consequatur omnis similique suscipit aliquam ad vel sapiente nesciunt dignissimos. Quo earum eos voluptatem commodi, repellat consequuntur voluptas in accusamus corporis aliquam sit vero assumenda distinctio dolorem exercitationem excepturi cupiditate enim quae!
+    </div> -->
```
Comment (oluwajuwon8) [inline index.html:60]: Hi Azeezat Omolola,
You have been doing a tremendous job so far 👌
To Highlight 🎉
GitHub flow is followed. ✔️
README is professional. ✔️

To Improve 🙏
 Please do not leave unused code as a comment in your codebase. Unused code as a comment makes your code not clean.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
+#main {
+  background-color: black;
+  font-family: arial, helvetica, sans-serif;
+  color: white;
+  text-align: center;
```
Comment (oluwajuwon8) [inline style.css:5]: Please stick to the design as much as possible (e.g., font, colours, font-family and margins)

**Review by oluwajuwon8 — COMMENTED** [review]
# Changes Required ♻

Please check my comment below and make all required changes.

Submit for Re-review when you are done.

Thanks😀

File: style.css (hunk 2 above)
Comment (bimbolabuari) [reply style.css:5]: Please always includes an explanation, a "why", for each requested change. For example:

This is a well-explained request: Please stick to the design as much as possible (e.g., font, colors, font family, and margins) as a mandate in the project requirement 👍 

This request lacks a "why": Please stick to the design as much as possible (e.g., font, colors, font-family, and margins)

**Review by bimbolabuari — CHANGES_REQUESTED** [review]
Hi  @oluwajuwon8 , 

You have done a good job so far 🚀  

## Highlights 🎉 

-  Use of emoji to make a comment look friendly 🎉 
-  You phrased the comments politely, as suggestions instead of orders 👍 

There are some issues that you still need to work on to go to the next project but you are almost there! 🎉 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

--------


_Reviewed by  @bimbolabuari_

File: style.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#main{
+  background-color: black;
+  font-family: arial, helvetica, sans-serif;
+  color:white;
+  Text-align: center;
```
Comment (oluwajuwon8) [inline style.css]: Please stick to the design as much as possible (e.g., font, colours, font family, and margins) as a mandate in the project requirement 👍

**Review by oluwajuwon8 — COMMENTED** [review]
# Required Changes ♻️
Check the comments under the review.

Thanks☺

**Review by rotimiazeez — APPROVED** [review]
# Hi @oluwajuwon8 👋, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**